modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/BookmarkDao.kt

```diff
@@ -63,6 +63,21 @@ abstract class BookmarkDao {
         deleted: Boolean = false,
     ): Flow<List<PodcastBookmark>>
 
+    @Query(
+        """SELECT bookmarks.*
+            FROM bookmarks
+            LEFT JOIN podcast_episodes ON bookmarks.episode_uuid = podcast_episodes.uuid 
+            WHERE bookmarks.podcast_uuid = :podcastUuid 
+            AND UPPER(bookmarks.title) LIKE UPPER(:title)
+            OR UPPER(podcast_episodes.title) LIKE UPPER(:title)
+            AND deleted = :deleted"""
```

Comment:
Would it be worth wrapping the OR in brackets as it seems confusing with both `AND` and `OR` commands together?
```suggestion
        """SELECT bookmarks.*
            FROM bookmarks
            LEFT JOIN podcast_episodes ON bookmarks.episode_uuid = podcast_episodes.uuid 
            WHERE bookmarks.podcast_uuid = :podcastUuid 
            AND (UPPER(bookmarks.title) LIKE UPPER(:title) OR UPPER(podcast_episodes.title) LIKE UPPER(:title))
            AND deleted = :deleted"""
```